unit/Makefile

```diff
@@ -1,19 +1,14 @@
-SRC = cpp_parser.cpp \
-      cpp_scanner.cpp \
-      elf_reader.cpp \
-      float_utils.cpp \
-      ieee_float.cpp \
-      json.cpp \
+CATCH_TESTS = catch_example.cpp \
+              util/sharing_map.cpp \
+              #Empty last line
+
+SRC = catch_entry_point.cpp \
       miniBDD.cpp \
-      osx_fat_reader.cpp \
```

Comment:
Unfortunately I don't have time to fix these at the moment, I have opened an issue to resolve this: #879. Would you prefer the files to be deleted (I'll attach them to the issue) so they aren't dangling?